script.js

```diff
@@ -1,43 +1,88 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] @team it's not good practice to disable the linter check for your file, kindly try and resolve the error instead of disabling linter checks.

Reply:
@danmainah please were asked to disable this if we encounter this issue
![image](https://user-images.githubusercontent.com/97846040/213009920-d3488b06-4c29-482c-a149-d7f825fc4777.png)